dask/dataframe/tests/test_io.py

```diff
@@ -547,11 +547,10 @@ def test_read_csv_of_modified_file_has_different_name():
     with filetext(text) as fn:
         mtime = os.path.getmtime(fn)
         a = read_csv(fn)
+        sleep(1)
         with open(fn, 'a') as f:
             f.write('\nGeorge,700')
             os.fsync(f)
-        while mtime == os.path.getmtime(fn):
-            sleep(0.01)
```

Comment:
Ah I see.  My assumption was that all changes are registered, they just take a while to percolate down.

Thoughts on how to manage the one second test issue?  Also what happens if the OS decides to act a bit more slowly than normal?  Is this test robust to very slow machines (like binstar-build)?